system-tests/projects/runner-e2e-specs/cypress-ct-webpack.config.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  numTestsKeptInMemory: 0,
+  video: false,
+  e2e: {
+    specPattern: 'no-matches/**.cy.*',
+  },
+  component: {
+    specPattern: 'cypress/e2e/**/*.cy.{js,jsx,ts,tsx}',
```

Comment:
Just to check; what's the value running all of these? Are they related to stack traces?

Reply:
They're not all being run, just specifying a pattern to allow those specs to be found and run individually in the system test. Stacktraces is specifically only running the set of specs under `cypress/e2e/errors/*` so I could make the pattern more specific but it shouldn't cause any harm to match more specs than needed since only a subset is actually being run